AppController/lib/cassandra_helper.rb

```diff
@@ -17,14 +17,18 @@
 
 
 # A String that indicates where we install Cassandra on this machine.
-CASSANDRA_DIR = "#{APPSCALE_HOME}/AppDB/cassandra"
+CASSANDRA_DIR = "/opt/cassandra"
```

Comment:
Do we need to persist the Cassandra binaries? I don't know why it should be different than the other system packages.